Xamarin.PropertyEditing.Mac/Controls/Custom/DynamicBox.cs

```diff
@@ -56,7 +56,7 @@ public string BorderColorName
 			{
 				this.borderColor = value;
 				if (value == null)
-					BorderColor = NSColor.Clear;
+					this.Layer.BorderColor = NSColor.Clear;
```

Comment:
```suggestion
					this.Layer.BorderColor = NSColor.Clear.CGColor;
```

Reply:
fixed - changed that after I commited :/